kolibri/content/utils/annotation.py

```diff
@@ -15,9 +15,9 @@ def _get_channel_list_from_scanning_contentdb_dir():
 
 def sync_channelmetadata():
```

Comment:
It's doing more than reading it though -- it's reading from the content DBs and writing the data into the cached table.

Perhaps `update_channel_metadata_cache`?